test/Test/Oscoin/DummyLedger.hs

```diff
@@ -17,14 +25,21 @@ import           Oscoin.Storage.Block.SQLite
 
 import           Codec.Serialise (Serialise)
 import qualified Data.ByteString as BS
+import qualified Data.ByteString.BaseN as BaseN
 
 import           Test.QuickCheck
 
 newtype DummyTx = DummyTx ByteString
```

Comment:
If it's 8 bytes, you could use a Word64 :man_shrugging:

Reply:
Using `Word64` would complicate the `TxRow` instance because we would need to convert the Word64 to and from a BytesString. We would also need to add `Hashable c Word64` to `IsCrypto` and implement the instances.